Source/Private/ASDisplayNodeInternal.h

```diff
@@ -170,27 +184,43 @@ AS_EXTERN NSString * const ASRenderingEngineDidDisplayNodesScheduledBeforeTimest
   /// Sentinel for layout data. Incremented when we get -setNeedsLayout / -invalidateCalculatedLayout.
   /// Starts at 1.
   std::atomic<NSUInteger> _layoutVersion;
-  
+
+
+  // Layout Spec performance measurement
+  ASDisplayNodePerformanceMeasurementOptions _measurementOptions;
+  NSTimeInterval _layoutSpecTotalTime;
+  NSInteger _layoutSpecNumberOfPasses;
+  NSTimeInterval _layoutComputationTotalTime;
+  NSInteger _layoutComputationNumberOfPasses;
+
+
```

Comment:
This seems like too much spacing. Since there's already a comment, should we keep the pattern of 1 blank line + the comment? At most 2 blank?